src/core/Form/MultiSelect/MultiSelectItem/MultiSelectItem.baseStyles.tsx

```diff
@@ -20,16 +20,11 @@ export const baseStyles = (theme: SuomifiTheme) => css`
       background-color: ${theme.colors.depthSecondaryDark1};
       color: ${theme.colors.blackBase};
 
-      & > .fi-multiselect-item_checkbox_icon {
-        position: absolute;
```

Comment:
Is this correct? Looks like selected state no longer applies position absolute the checkbox icon, or was this redundant?

Reply:
Seemed redundant to me since it's defined on line 57 in the "base styles" for the icon class.

Reply:
So it seems, looks like these selected styles for the actual icon might be redundant as the icon is never there when an item is not checked, but it should not be a concern for this PR.